src/projects_parser.py

```diff
@@ -110,6 +110,37 @@ def canonicalize_projects(projects, **kwargs):
         canonical_projects.append(canon)
     return canonical_projects
 
+def combine_fields(project, new_field, field1, field2, deletes=None):
```

Comment:
You can combine two fields into one with this function. They'll be joined with a comma.